### PR TITLE
Update WikiANN data URL

### LINKED ISSUE #2691: xtreme / pan-x cannot be downloaded
## Describe the bug

Dataset xtreme / pan-x cannot be loaded

Seems related to https://github.com/huggingface/datasets/pull/2326

## Steps to reproduce the bug

```python
dataset = load_dataset("xtreme", "PAN-X.fr")
```

## Expected results

Load the dataset

## Actual results

```
FileNotFoundError: Couldn't find file at https://www.dropbox.com/s/12h3qqog6q4bjve/panx_dataset.tar?dl=1
```

## Environment info

- `datasets` version: 1.9.0
- Platform: macOS-11.4-x86_64-i386-64bit
- Python version: 3.8.11
- PyArrow version: 4.0.1

### PR DESCRIPTION
**Pull request by albertvillanova**
WikiANN data source URL is no longer accessible: 404 error from Dropbox.

We have decided to host it at Hugging Face. This PR updates the data source URL, the metadata JSON file and the dataset card.

Close #2691.

### CODE REVIEW

File: datasets/wikiann/wikiann.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 _DESCRIPTION = """WikiANN (sometimes called PAN-X) is a multilingual named entity recognition dataset consisting of Wikipedia articles annotated with LOC (location), PER (person), and ORG (organisation) tags in the IOB2 format. This version corresponds to the balanced train, dev, and test splits of Rahimi et al. (2019), which supports 176 of the 282 languages from the original WikiANN corpus."""
 
 # use ?dl=1 to force browser to download data instead of displaying it
-_DATA_URL = "https://www.dropbox.com/s/12h3qqog6q4bjve/panx_dataset.tar?dl=1"
+_DATA_URL = "https://s3.amazonaws.com/datasets.huggingface.co/wikiann/1.1.0/panx_dataset.zip"
```
Comment (julien-c) [inline datasets/wikiann/wikiann.py:46]: i guess you can then remove the comment line above this one

Comment (albertvillanova) [reply datasets/wikiann/wikiann.py:46]: Yes, you are right @julien-c 😅

**Review by lhoestq — APPROVED** [review]
Thanks !

**Comment by lhoestq** [comment]
We have to update the URL in the XTREME benchmark as well:

https://github.com/huggingface/datasets/blob/0dfc639cec450ed8762a997789a2ed63e63cdcf2/datasets/xtreme/xtreme.py#L411-L411

**Pull request merged**